packages/kit/src/core/dev/index.js

```diff
@@ -177,7 +177,15 @@ class Watcher extends EventEmitter {
 
 		this.vite = await vite.createServer(merged_config);
 
-		handler = await create_handler(this.vite, this.config, this.dir, this.cwd, this.manifest);
+		const getManifest = () => {
+			if (!this.manifest) {
+				throw new Error("Shouldn't clear manifest after server initialized");
+			}
```

Comment:
Will there be a situation where this will eventually throw?

Reply:
Doesn't see a current path that it will throw. It should only be undefined before the init method call. It's more likely to throw in line 92. It's just because init is async so we can't enforce it to be not nullable unless we have an empty object in the constructor.